### DISCUSSION TITLE
Command to do "Fast Database Reboot"

### DISCUSSION THREAD

**Discussion by dishant26** [supabase/supabase#30530] [General]
I came across a way to restart my database quickly on the Supabase UI by using "Fast Database Reboot" instead of "Restart Project".
Is there a command to do this?

![image](https://github.com/user-attachments/assets/eb79e9d4-b794-4a2f-9e20-b6bd5fec2fca)

**Comment by farhannkaskar** [comment]
Any update here Supbase team? Can someone explain what is fast restart? I don't see any neat documentation on it

**Comment by Jonny1987** [comment]
Same...please can someone explain?
